lib/reek/cli/command/todo_list_command.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'commandable'
+require_relative '../../examiner'
+
+module Reek
+  module CLI
+    module Command
+      #
+      # A command to collect smells from a set of sources and writes a configuration
+      # file that can serve as a todo list.
+      #
+      class TodoListCommand
+        include Commandable
+
+        def execute(app)
+          scan_for_smells app
+          File.write '.todo.reek', Hash[groups].to_yaml
+          puts "\n'.todo.reek' generated! You can now use "\
+                  'this as a starting point for your configuration.'
+        end
+
+        private
+
+        def groups
+          @groups ||= reporter.smells.group_by(&:smell_type).map do |type, smells|
+            [type, { 'exclude' => smells.map(&:context) }]
+          end
+        end
+
+        def reporter
+          @reporter ||= Report.report_class(:yaml).new
```

Comment:
It's indeed not important. Actually now that I think of it this is highly confusing. 
Let me refactor this.